### PR TITLE
[docs]: Docker context commands

### PR DESCRIPTION
**Pull request by dmakeroam**
docs: add docker context commands

### CODE REVIEW

File: content/faq/docker/docker-context.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Docker context
+showMetadata: true
+editable: true
+showToc: true
+---
+
```
Comment (aaronamm) [inline content/faq/docker/docker-context.md:7]: Could you please a little introduction here to explain what is docker context?
Thanks.

**Comment by aaronamm** [comment]
It looks like your later PR is based on a previous PR which is not right for this situation.

Actually, you need to have each of your PR based on the main branch and keep only one changed file for each PR.
This is because each of your PR targeting to change only one docker-compose content.

Please let me know if you would like to reset this PR to have only one changed file which is relevant to this PR.
If not, I think we need to start merging from the first PR then changes in other files will be removed from this PR automatically.

**Comment by aaronamm** [comment]
```sh
$ git remote -v
```
You should get:
```sh
upstream        git@github.com:dotnetthailand/dotnetthailand.github.io.git
```
If not, please run:
```sh
$ git remote set-url upstream git@github.com:dotnetthailand/dotnetthailand.github.io.git
```
Then:
```sh
$ git fetch upstream
$ git checkout docs/docker-context
$ git rebase upstream/main
```
Then `git log --oneline` and you should get:
```sh
f076dfe4 (HEAD -> docs/docker-context) docs: add docker context commands
8378cad4 docs: improve security and image size
d6e73135 (upstream/main, main) Merge pull request #124 from dotnetthailand/docs/mysql-secrets 😍😍😍
```
Run `git status` to check all changed and staged files. I think you need only these files:
```
modified:   content/faq/docker.md
new file:   content/faq/docker/docker-context.md
```

Reset another file:
```sh
$ git reset content/web-frameworks/django/docker-compose-for-django-mysql.md
$git checkout content/web-frameworks/django/docker-compose-for-django-mysql.md
```

Commit and force push
```sh
git commit -m "Add Docker context commands"
git push -f upstream docs/docker-context
```

File: content/faq/docker/docker-context.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+# Docker Context
+A single Docker CLI can have multiple contexts. Each context contains all of the endpoint and security information required to manage a different cloud environment, Swarm nodes, or Kubernetes nodes. The docker context command makes it easy to configure these contexts and switch between them.
+
```
Comment (aaronamm) [inline content/faq/docker/docker-context.md:10]: @dmakeroam this is cool, thanks.

Comment (mildronize) [reply content/faq/docker/docker-context.md:10]: Look good @dmakeroam

**Review by aaronamm — APPROVED** [review]
Nice work, thanks for your contribution.

**Review by mildronize — APPROVED** [review]
Thank you for contribution. I appreciate your effort.

**Pull request merged**